tests/ui/lang/core/ptr/allocate_const_scalar.rs

```diff
@@ -0,0 +1,18 @@
+// Doesn't work, only worked before because I think it got optimized away before
+// hitting the backend.
+
+// Needed because compiletest currently emits `\r\n` on Windows
+// normalize-stderr-test "\r\n" -> "\n"
```

Comment:
Hmm, how sure are we that this isn't caused by SPIRV-Tools? The Rust side should not be using `\r` at all.

Reply:
That is a possibility, but I would have assumed `compiletest` would normalise the all of the `stderr` output.